runtime/tests/interpreter/resources_test.go

```diff
@@ -2520,34 +2520,32 @@ func TestInterpretResourceDestroyedInPreCondition(t *testing.T) {
                 }
             }
         }
-
```

Comment:
Maybe leave the line breaks between the declarations, otherwise the code becomes quite unreadable